packages/psss/src/AppProviders.js

```diff
@@ -5,21 +5,26 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Provider } from 'react-redux';
+import { QueryClient, QueryClientProvider } from 'react-query';
 import { MuiThemeProvider, StylesProvider } from '@material-ui/core/styles';
 import { ThemeProvider } from 'styled-components';
 import CssBaseline from '@material-ui/core/CssBaseline';
 import { PersistGateProvider } from './utils/PersistGateProvider';
 import { theme } from './theme';
 
+const queryClient = new QueryClient();
+
 export const AppProviders = ({ children, store }) => {
   return (
     <Provider store={store}>
       <PersistGateProvider store={store}>
         <StylesProvider injectFirst>
           <MuiThemeProvider theme={theme}>
             <ThemeProvider theme={theme}>
-              <CssBaseline />
-              {children}
+              <QueryClientProvider client={queryClient}>
```

Comment:
Version 3 requires a `QueryClientProvider`, see: https://tanstack.com/query/v4/docs/react/guides/migrating-to-react-query-3#the-querycache-has-been-split-into-a-queryclient-and-lower-level-querycache-and-mutationcache-instances